.github/workflows/ci.yml

```diff
@@ -123,6 +125,9 @@ jobs:
           # See https://clang.llvm.org/docs/UndefinedBehaviorSanitizer.html
           UBSAN_OPTIONS: print_stacktrace=1
 
+      - name: Benchmarks
```

Comment:
Minor thing, but in the rest of the GitHub Action definition we don't put names in the steps, as they are short enough to understand?

Reply:
I definetly prefer names. For instance `cmake --build ./build ...` looks quite long and unreadable to me hahah

Reply:
Right. Maybe. Can you avoid the title now for consistency? You can add the titles for all the steps in another PR if we want and we discuss there?